File: N-Tier/AspCoreProject/softweb-common/Models/UserDto.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Softweb.Models
{
    public class UserDto
    {
        public int Id { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }
        public UserDto() { }
        public UserDto(int id, string firstname, string lastname)
        {
            this.Id = id;
            this.FirstName = firstname;
            this.LastName = lastname;
        }

    }
}

File: N-Tier/JavaProject/softweb-common/src/main/java/com/softweb/UserDto.java
package com.softweb;


import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@Builder
//@AllArgsConstructor
@NoArgsConstructor
public class UserDto {

    @JsonProperty("idddd")
    private int id;
    @JsonProperty("firstName")
    private String firstName;
    @JsonProperty("lastName")
    private String lastName;

    public UserDto(int _id, String _firstName, String _lastName) {
        this.id = _id;
        this.firstName = _firstName;
        this.lastName = _lastName;
    }

}
